db/migrate/20211103120946_clean_emails_footer.rb

```diff
@@ -0,0 +1,30 @@
+class CleanEmailsFooter < ActiveRecord::Migration[6.1]
+  def up
+    Setting.reset_column_information
+    Setting.emails_footer = Setting
```

Comment:
This will always set the value even if there was no `emails_footer` setting before. I don't see an immediate harm in doing it but we would maintain more flexibility if that value was only saved in case there had been one before.

Reply:
Good point, fixed